core/core-workflow-steps/src/main/java/org/eclipse/sw360/antenna/workflow/analyzers/CsvAnalyzer.java

```diff
@@ -100,4 +112,34 @@ public WorkflowStepResult yield() throws AntennaException {
     public String getName() {
         return "CSV";
     }
+
+    private License createLicense(String short_name, String long_name) {
+        License license = new License();
+        license.setName(short_name);
+        license.setLongName(long_name);
+
+        return license;
+    }
+
+    private void mapCoordinate(Artifact artifact, CSVRecord record) {
```

Comment:
instead of this style of coding with mutations it would maybe be more consistent to use the same pattern as for `createLicense` e.g. 
```suggestion
    private FactOrSomeOtherMatchingClass mapCoordinate(CSVRecord record) {
```